### ISSUE TITLE
materials-editor.ui and libFreeCAD_widgets.so installed to the wrong directory

### ISSUE THREAD

**Issue by jffmichi** [FreeCAD/FreeCAD#15571]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I'm building on Arch Linux in the directory `/home/jffmichi/.cache/pacaur/freecad-git` using `makepkg` which in turn calls
```
DESTDIR="/home/jffmichi/.cache/pacaur/freecad-git/pkg/freecad-git" ninja -C build install
```

This is supposed to install freecad into `$DESTDIR`. makepkg then builds the Arch Linux package from that directory. However there is an issue where two files are installed into the wrong directory:

`materials-editor.ui` is installed into the something like `$DESTDIR/$SRCDIR`:
```
Installing: /home/jffmichi/.cache/pacaur/freecad-git/pkg/freecad-git/home/jffmichi/.cache/pacaur/freecad-git/src/build/Mod/Material/Resources/ui/materials-editor.ui
```

Edit: on closer inspection, `materials-editor.ui` should probably not be installed at all since it is included in [Material.qrc](https://github.com/FreeCAD/FreeCAD/blob/main/src/Mod/Material/Gui/Resources/Material.qrc). The same goes for the other files in `Material.qrc` but those are installed to a valid directory.

`libFreeCAD_widgets.so` is installed into something like `$DESTDIR/$DESTDIR`:
```
Installing: /home/jffmichi/.cache/pacaur/freecad-git/pkg/freecad-git/home/jffmichi/.cache/pacaur/freecad-git/pkg/freecad-git/usr/lib/qt6/plugins/designer/libFreeCAD_widgets.so
```

should be:
```
Installing: /home/jffmichi/.cache/pacaur/freecad-git/pkg/freecad-git/usr/lib/qt6/plugins/designer/libFreeCAD_widgets.so
```

### Full version info

```shell
commit 187296d
```

### Subproject(s) affected?

Other (specify in description)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jffmichi** [comment]
Oh it seems the `materials-editor.ui` issue might already be addressed in the pull request #15565

**Comment by luzpaz** [comment]
@adrianinsaval is this addressed in #15565 ?

**Comment by pkreussel** [comment]
At least,  #15565 prevents `cmake --install` from trying to install `materials-editor.ui` in the Home-Dir, trying to overwrite the original source file. 

My built package installs (also on Arch Linux, but the problem would arise in other package build systems, too),  FreeCad starts and seems to work normally.

I have not use the compiled programm a lot, though, cannot say if a file might be missing.

**Comment by davesrocketshop** [comment]
This pr should be closed in favour of pr #16322
